### DISCUSSION TITLE
Show Episode number in Seasons view

### DISCUSSION THREAD

**Discussion by Pfuenzle** [jellyfin/jellyfin-vue#1441] [UI & UX]
Currently, when looking at all the Episodes of a season, it's only showing the episode name and a description, but not an Episode number. This gets more and more confusing, the more episodes a season has, as you have to count the episodes to get the one you want to watch. 
Is an episode counter already on the To-do list?
I'm not experienced with Vue, but I think adding an counter should not be that extensive

**Comment by ThibaultNocchi** [comment]
Hello,
Yeah that's a needed improvement. Tbh I think it'd be better suited as an issue, discussions are usually used for matters which need a little more thoughts :)

**Comment by doc-code-hub** [comment]
I misread this discussion and made this fix #1682, but I still find it useful.

This is to see the season and episode number, when on the **single episode** page

I hope it will be included  :)
 
![image](https://user-images.githubusercontent.com/61033694/148667877-90447928-bb2e-4888-9020-5945d5a1bd01.png)
